mteb/abstasks/AbsTask.py

```diff
@@ -51,6 +52,15 @@ def _multilabel_subsampling(
     return dataset_dict
 
 
+class AbsDescriptiveStatistics(TypedDict):
+    """Abstract class for descriptive statistics.
+
+    num_samples: number of samples in the dataset.
+    """
+
+    num_samples: int
```

Comment:
Would just do:

```suggestion
class DescriptiveStatistics(TypedDict):
    """Class for descriptive statistics.

    num_samples: number of samples in the dataset.
    """

    num_samples: int
```

Unsure if we even need num_samples specified here, but it all cases use it then it is fine.